Make btns buttons, inherit font family

alanmoo opened the pull request:
Normalize was resetting font family of buttons to a general sans-serif. This still has theoretical problems of inheriting fonts for specific sections, but I don’t know that it’s going to be a realistic issue for us. The other option was to set the font-family of those elements to the font variable string we use, but I don't feel like it really makes a difference, as that can lead to cascade override issues later on.

Also makes the index page use `<button>` instead of the fictional `<btn>`

Closes #102

gvn commented:
Can you also fix this so we retain our spacing between buttons in the demo? (Or switch to `mb...` classes)

https://github.com/mozilla/mofo-bootstrap/blob/master/src/scss/demo.scss#L79

gvn commented:
Also a couple other buttons seem to have lost their styles.

alanmoo commented:
I updated that style and removed the buttons that were having issues from the demo page, because they weren't actually buttons that are built into bootstrap.

gvn commented:
Can you also remove that CSS targeting `btn` that I linked above?

alanmoo commented:
I changed it to `.btn`. You want it gone instead?

gvn commented:
Oh, NM
